src/core/vector/qgsvectorlayertemporalproperties.cpp

```diff
@@ -442,11 +442,22 @@ QString dateTimeExpressionLiteral( const QDateTime &datetime )
          .arg( datetime.time().second() + datetime.time().msec() / 1000.0 );
 }
 
-QString QgsVectorLayerTemporalProperties::createFilterString( QgsVectorLayerTemporalContext, const QgsDateTimeRange &filterRange ) const
+QString QgsVectorLayerTemporalProperties::createFilterString( QgsVectorLayerTemporalContext context, const QgsDateTimeRange &filterRange ) const
```

Comment:
@elpaso do you know why the `context` is not a const ref in this case?

Reply:
mmm, no but I'll check if we can make it const ref (or would it be an API change?)

Reply:
done

Reply:
Nice thanks